compiler/noirc_frontend/src/parser/parser/expression.rs

```diff
@@ -109,15 +136,19 @@ impl Parser<'_> {
         }
     }
 
-    /// AtomOrUnaryRightExpression
-    ///     = Atom UnaryRightExpression*
-    fn parse_atom_or_unary_right(&mut self, allow_constructors: bool) -> Option<Expression> {
+    /// AtomOrMemberAccessOrCallOrMethodCallOrIndexExpression
+    ///     = Atom MemberAccessOrCallOrMethodCallOrIndexExpression*
+    fn parse_atom_or_member_access_or_call_or_method_call_or_index(
```

Comment:
```suggestion
    /// Atom = Quark AtomRhs*
    fn parse_atom(
```
Alternatively we can use `high_prec_postfix_op` or similar to distinguish it from `Cast` which is lower prec

Reply:
Done!